___Execucao_com 2 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 58.045296390075237
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:58
	Energy:	0.0042 kWh
	CO2eq:	2.0078 g
	This is equivalent to:
	0.0187 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:58
	Energy:	0.0042 kWh
	CO2eq:	2.0078 g
	This is equivalent to:
	0.0187 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 2 Treads_concluida___
___Execucao_com 4 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 55.589647079817951
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0041 kWh
	CO2eq:	1.9615 g
	This is equivalent to:
	0.0182 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0041 kWh
	CO2eq:	1.9615 g
	This is equivalent to:
	0.0182 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 4 Treads_concluida___
___Execucao_com 6 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 55.245146416127682
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0042 kWh
	CO2eq:	1.9929 g
	This is equivalent to:
	0.0185 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0042 kWh
	CO2eq:	1.9929 g
	This is equivalent to:
	0.0185 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 6 Treads_concluida___
___Execucao_com 8 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 54.757933336310089
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0043 kWh
	CO2eq:	2.0258 g
	This is equivalent to:
	0.0188 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0043 kWh
	CO2eq:	2.0258 g
	This is equivalent to:
	0.0188 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 8 Treads_concluida___
___Execucao_com 10 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 55.236151715740561
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0044 kWh
	CO2eq:	2.0978 g
	This is equivalent to:
	0.0195 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0042 kWh
	CO2eq:	1.9926 g
	This is equivalent to:
	0.0185 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 10 Treads_concluida___
___Execucao_com 12 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 55.918966343160719
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0045 kWh
	CO2eq:	2.1595 g
	This is equivalent to:
	0.0201 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0045 kWh
	CO2eq:	2.1303 g
	This is equivalent to:
	0.0198 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 12 Treads_concluida___
___Execucao_com 14 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 55.790120020043105
CarbonTracker: WARNING - Failed to retrieve carbon intensity: Defaulting to average carbon intensity 475 gCO2/kWh.
CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: Porto Alegre, Rio Grande do Sul, BR.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0046 kWh
	CO2eq:	2.1719 g
	This is equivalent to:
	0.0202 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0046 kWh
	CO2eq:	2.1719 g
	This is equivalent to:
	0.0202 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 14 Treads_concluida___
___Execucao_com 16 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 53.676483856048435
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:54
	Energy:	0.0045 kWh
	CO2eq:	2.1432 g
	This is equivalent to:
	0.0199 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:54
	Energy:	0.0045 kWh
	CO2eq:	2.1432 g
	This is equivalent to:
	0.0199 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 16 Treads_concluida___
___Execucao_com 18 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 51.073812282178551
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:51
	Energy:	0.0045 kWh
	CO2eq:	2.1180 g
	This is equivalent to:
	0.0197 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:51
	Energy:	0.0045 kWh
	CO2eq:	2.1180 g
	This is equivalent to:
	0.0197 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 18 Treads_concluida___
___Execucao_com 20 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 50.968047449830920
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:51
	Energy:	0.0045 kWh
	CO2eq:	2.1541 g
	This is equivalent to:
	0.0200 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:51
	Energy:	0.0045 kWh
	CO2eq:	2.1541 g
	This is equivalent to:
	0.0200 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 20 Treads_concluida___
___Execucao_com 22 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 52.987960993777961
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:53
	Energy:	0.0049 kWh
	CO2eq:	2.3323 g
	This is equivalent to:
	0.0217 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:53
	Energy:	0.0049 kWh
	CO2eq:	2.3323 g
	This is equivalent to:
	0.0217 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 22 Treads_concluida___
___Execucao_com 24 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 55.103601485956460
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0048 kWh
	CO2eq:	2.2863 g
	This is equivalent to:
	0.0213 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0048 kWh
	CO2eq:	2.2863 g
	This is equivalent to:
	0.0213 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 24 Treads_concluida___
___Execucao_com 26 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 51.490354273933917
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:52
	Energy:	0.0047 kWh
	CO2eq:	2.2168 g
	This is equivalent to:
	0.0206 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:52
	Energy:	0.0047 kWh
	CO2eq:	2.2168 g
	This is equivalent to:
	0.0206 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 26 Treads_concluida___
___Execucao_com 28 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 54.509824742097408
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0047 kWh
	CO2eq:	2.2332 g
	This is equivalent to:
	0.0208 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0047 kWh
	CO2eq:	2.2332 g
	This is equivalent to:
	0.0208 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 28 Treads_concluida___
___Execucao_com 30 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 54.520792024210095
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0050 kWh
	CO2eq:	2.3674 g
	This is equivalent to:
	0.0220 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:55
	Energy:	0.0050 kWh
	CO2eq:	2.3674 g
	This is equivalent to:
	0.0220 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 30 Treads_concluida___
___Execucao_com 32 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 53.606115814298391
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:54
	Energy:	0.0047 kWh
	CO2eq:	2.2546 g
	This is equivalent to:
	0.0210 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:54
	Energy:	0.0047 kWh
	CO2eq:	2.2546 g
	This is equivalent to:
	0.0210 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 32 Treads_concluida___
___Execucao_com 34 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 51.968664838932455
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:52
	Energy:	0.0047 kWh
	CO2eq:	2.2175 g
	This is equivalent to:
	0.0206 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:52
	Energy:	0.0047 kWh
	CO2eq:	2.2175 g
	This is equivalent to:
	0.0206 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 34 Treads_concluida___
___Execucao_com 36 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 51.898796714842319
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:52
	Energy:	0.0047 kWh
	CO2eq:	2.2308 g
	This is equivalent to:
	0.0208 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:52
	Energy:	0.0047 kWh
	CO2eq:	2.2308 g
	This is equivalent to:
	0.0208 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 36 Treads_concluida___
___Execucao_com 38 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 56.506190947722644
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:57
	Energy:	0.0051 kWh
	CO2eq:	2.4158 g
	This is equivalent to:
	0.0225 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:57
	Energy:	0.0051 kWh
	CO2eq:	2.4158 g
	This is equivalent to:
	0.0225 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 38 Treads_concluida___
___Execucao_com 40 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 55.552846128121018
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0050 kWh
	CO2eq:	2.3763 g
	This is equivalent to:
	0.0221 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0050 kWh
	CO2eq:	2.3763 g
	This is equivalent to:
	0.0221 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 40 Treads_concluida___
___Execucao_com 42 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 55.574073519092053
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0050 kWh
	CO2eq:	2.3887 g
	This is equivalent to:
	0.0222 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:56
	Energy:	0.0050 kWh
	CO2eq:	2.3887 g
	This is equivalent to:
	0.0222 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 42 Treads_concluida___
___Execucao_com 44 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 49.718601132277399
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:50
	Energy:	0.0046 kWh
	CO2eq:	2.1861 g
	This is equivalent to:
	0.0203 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:50
	Energy:	0.0046 kWh
	CO2eq:	2.1861 g
	This is equivalent to:
	0.0203 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 44 Treads_concluida___
___Execucao_com 46 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 50.613418381195515
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:51
	Energy:	0.0047 kWh
	CO2eq:	2.2198 g
	This is equivalent to:
	0.0206 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:51
	Energy:	0.0047 kWh
	CO2eq:	2.2198 g
	This is equivalent to:
	0.0206 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 46 Treads_concluida___
___Execucao_com 48 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.


FFT_SERIAL
  C version

  Demonstrate an implementation of the Fast Fourier Transform
  of a complex data vector.

  Accuracy check:

    FFT ( FFT ( X(1:N) ) ) == N * X(1:N)

             N      NITS    Error

             2     10000  7.859082e-17
             4     10000  1.209837e-16
             8     10000  6.820795e-17
            16     10000  1.438671e-16
            32      1000  1.331210e-16
            64      1000  1.776545e-16
           128      1000  1.929043e-16
           256      1000  2.092319e-16
           512       100  1.927488e-16
          1024       100  2.312093e-16
          2048       100  2.445006e-16
          4096       100  2.476589e-16
          8192        10  2.571250e-16
         16384        10  2.736298e-16
         32768        10  2.924127e-16
         65536        10  2.833553e-16
        131072         1  3.142312e-16
        262144         1  3.216005e-16
        524288         1  3.282664e-16
       1048576         1  3.284479e-16
       2097152         1  3.509548e-16
       4194304         1  3.560792e-16
       8388608         1  3.709655e-16
      16777216         1  3.651822e-16
      33554432         1  3.816475e-16
      67108864         1  4.072041e-16
Execution Time: 50.281132248230278
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:50
	Energy:	0.0047 kWh
	CO2eq:	2.2268 g
	This is equivalent to:
	0.0207 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:50
	Energy:	0.0047 kWh
	CO2eq:	2.2268 g
	This is equivalent to:
	0.0207 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 48 Treads_concluida___
O arquivo 'log_bruto.txt' não foi encontrado.
